styles.css

```diff
@@ -0,0 +1,971 @@
+.container-space-around {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	flex-wrap: wrap;
+	width: 100%;
+}
+
+li {
+	list-style: none;
+}
+
+a {
+	text-decoration: none;
+	color: inherit;
+}
+
+button {
+	cursor: pointer;
+	text-align: center;
+	outline: none;
+}
+
+input {
+	outline: none;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+	margin: 0.5rem;
+}
+
+/*navbar CSS*/
+.navbar {
+	background: white;
+	padding: 0.5rem 1rem;
+	display: flex;
+	width: 100%;
+	height: 3rem;
+	box-shadow: 1px 2px 5px var(--color-grey-bg);
+	position: fixed;
+	top: 0;
+	left: 0;
+	z-index: 10;
+	box-sizing: border-box;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+}
+
+.hide-menu {
+	transition: all 0.5s;
+	opacity: 0%;
+	transform: translateX(-100%);
+}
+
+.show-menu {
+	transition: all 0.5s;
+	opacity: 100%;
+	transform: translateX(0%);
+}
+
+.menu {
+	position: absolute;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	width: 100%;
+	height: 100vh;
+	overflow-y: auto;
+	top: 3rem;
+	left: 0;
+	background-color: var(--color-grey-light);
+}
+
+.menu-item {
+	display: inline;
+	margin-right: 1rem;
+	padding: 1rem;
+	width: 100%;
+	padding-left: 3rem;
+	text-align: left;
+	border-bottom: 1px solid white;
+	transition: padding 1s;
+}
+
+.menu-item a{
+	display: block;
+}
+
+.menu-item:hover {
+	background-color: var(--color-primary-bg);
+	color: var(--color-primary-dark);
+	padding-left: 5rem;
+}
+
+.brand-logo {
+	margin-left: 1rem;
+}
+.brand-name {
+	font-weight: 600;
+	font-size: 1.5rem;
+	font-family: "Dancing Script";
+	color: var(--color-primary);
+}
+
+.brand-name span {
+	font-weight: 500;
+	font-size: 1.2rem;
+	font-family: "Dancing Script";
+	color: var(--color-primary-dark);
+}
+
+.brand-logo img {
+	height: 2rem;
+	width: 2rem;
+}
+
+.hide-in-main-nav{
+	visibility: hidden;
+	width: 0%;
+}
+.hide-in-side-nav{
+	display: auto;
+}
+
+.hamburger-menu-line {
+	display: block;
+	width: 2rem;
+	height: 0.2rem;
+	background: var(--color-primary);
+	margin: 0.3rem 0.2rem;
+	border-radius: 5px;
+	transition: all 500ms linear;
+	transform-origin: 5px 2px;
+}
+
+.click .hamburger-menu-line:first-child {
+	transform: rotate(45deg);
+}
+
+.click .hamburger-menu-line:nth-child(2) {
+	opacity: 0;
+}
+
+.click .hamburger-menu-line:nth-child(3) {
+	transform: rotate(-45deg);
+}
+
+/* Alerts */
+
+.alert {
+	width: 90%;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding: 0.5rem 1rem;
+	border-radius: 0.2rem;
+	margin: 0.2rem;
+}
+
+.alert div {
+	margin-right: 1.5rem;
+	width: auto;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+.alert .ic {
+	color: inherit;
+	width: 3rem;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	font-size: 0.6rem;
+}
+
+.alert button {
+	border: none;
+	padding: 0;
+	font-size: 1rem;
+	font-weight: 700;
+	border-radius: 50%;
+	width: 1.5rem;
+	height: 1.5rem;
+}
+
+.alert-success,
```

Comment:
Don't need to select `alert-success` twice directly change the button property of that alert, remove line 270 and do it for all other alerts as well.

Reply:
Actually, I want to assign properties to Alert as well as Button so it's done in that way. Also, don't want the user to use extra class for button too.